templates/documents/copy.html.twig

```diff
@@ -0,0 +1,26 @@
+{% extends 'layout-with-submenu.html.twig' %}
+
+{% block title %}{% trans %}Copy document to other cases{% endtrans %}{% endblock %}
+
+{% block header %}
+    <div class="row">
+        <div class="col">
+            <h1>{% trans %}Basic information{% endtrans %}</h1>
+        </div>
+        <div class="col-auto">
+            <a name="" id="" class="btn btn-secondary" href="{{ url('document_index', {id: case.id}) }}" role="button">{% trans %}Cancel{% endtrans %}</a>
+        </div>
+    </div>
+
+    {{ form_start(copy_document_form) }}
+    <div class="form-row">
+        <div class="col-md-2 mb-3">
+            <label >{% trans %}Cases{% endtrans %}</label>
+            {{ form_widget(copy_document_form.cases, {'attr': {'class': 'form-control', 'required': true}}) }}
+        </div>
+    </div>
+    {{ form_widget(copy_document_form.save, {'attr': {'class': 'btn btn-primary', 'type': 'submit'}}) }}
+    {{ form_rest(copy_document_form) }}
```

Comment:
If you use both `form_rest` and `form_end` does it add two `</form>` elements?

Reply:
Removed unnecessary `form_rest` - `form_end` calls `form_rest`.